src/utils/types-utils.ts

```diff
@@ -6,12 +6,14 @@ import { CoreUtils } from '@laserfiche/lf-js-utils';
 export function convertApiToLfFieldInfo(val: WFieldInfo): LfFieldInfo {
   const id = CoreUtils.validateDefined(val.id, 'id');
   const name = CoreUtils.validateDefined(val.name, 'name')!;
+  const displayName: string = val.displayName ?? name;
```

Comment:
should we use CoreUtils.validateDefined here for displayName as well?

Reply:
updated, forgot we had a version with the defaultValue fallback